docs/codeReview.md

```diff
@@ -5,13 +5,27 @@ Review progress is tracked in the [Sprint Board](https://github.com/orgs/Parabol
 If there is no issue belonging to the PR, then the PR itself should be added to the board.
 A PR advances from self-review to reviewer-review and finally maintainer review before it can get merged.
 
-## General rules
+## Submitting a PR
 
 - Once you're assigned to a task, it's your responsibility to split it the way you can submit reasonably sized PRs. It’s ok to have multiple PRs for a single issue.
 - Use draft PRs to ask for early feedback.
-- It should be a reviewer’s goal to do the code review within 1 business day. If you’re unable to do it, leave a comment.
-- If you’re busy it’s ok to assign someone else as a reviewer.
 - When submitting a PR, provide context. What's the task? What's changed? Why? If you can, record a Loom to share why things are done that way. It'll be easier for a reviewer to understand what decision you've made and why.
+
+## Requesting Code Reviews
+
+- Self-assess which domains are involved in the changes made by your PR, find a reviewer from among the folks who fill the [Code Reviewer (L2–3)](https://www.notion.so/Code-Reviewer-L2-3-a47bb0759a0b41b5b0469ff14a8cdaae) role, paying specific attention to who GitHub suggests, who has indicated expertise within a domain (e.g. backend, front-end, etc.), and who has capacity to review within 24 hours.
+- Consider requesting review from a group of reviewers at once
+- Post a link in a slack channel: [**#t_product_review_requests**](https://app.slack.com/client/T08FL6336/C043A70QXTL) to allow anyone interested to review this PR earlier than assigned Code Reviewer (unless you don't need one particular person)
+- If the PR has not been reviewed within 24 hours, send a reminder to Slack or request a review from another reviewer or group of reviewers, or consider adding an agenda item to the [Parabol – Product](https://www.notion.so/Parabol-Product-1065acffc95d4f64a71b7808bad98ff5) check-in meeting
+- If the developer creating the PR feels that maintainer review isn't necessary, they should add a label: “One Review Required”. Use the label for PRs of any size and complexity If you are confident enough and it feels safe.
+
+## Doing Code Reviews
+
+- Doing code reviews and unblocking others is also important part of your job. It is totally fine that sometimes you may do more PR reviews than coding.
+- Prioritize unblocking others by starting working day by doing [code reviews](https://github.com/pulls/review-requested)
+- Try to complete PR reviews within 24 hours
```

Comment:
+1 I would rephrase it to within 1 working day

Reply:
Rephrased